packages/devtools/lib/src/debugger/debugger_state.dart

```diff
@@ -20,29 +20,27 @@ class DebuggerState {
 
   final Map<String, Script> _scriptCache = <String, Script>{};
 
-  final BehaviorSubject<bool> _paused = BehaviorSubject<bool>.seeded(false);
-  final BehaviorSubject<bool> _supportsStepping =
-      BehaviorSubject<bool>.seeded(false);
+  final _paused = _Property<bool>(false);
+  final _supportsStepping = _Property<bool>(false);
 
   Event lastEvent;
 
-  final BehaviorSubject<List<Breakpoint>> _breakpoints =
-      BehaviorSubject<List<Breakpoint>>.seeded(<Breakpoint>[]);
+  final _breakpoints = _Property<List<Breakpoint>>([]);
 
-  final BehaviorSubject<String> _exceptionPauseMode = BehaviorSubject();
+  final _exceptionPauseMode = _Property<String>();
 
   InstanceRef _reportedException;
 
   bool get isPaused => _paused.value;
 
-  Stream<bool> get onPausedChanged => _paused;
+  Stream<bool> get onPausedChanged => _paused.stream;
 
   Stream<bool> get onSupportsStepping =>
-      Observable<bool>.concat(<Stream<bool>>[_paused, _supportsStepping]);
+      StreamGroup.merge([_paused.stream, _supportsStepping.stream]);
```

Comment:
I don't know the semantics of `StreamGroup.merge` - does it return the logical equivalent of `isPaused && supportsStepping` (firing events each time the boolean value would change)?

Reply:
No, it emits either value as they come through. I have a change switching to `ValueNotifier` which also fixes these semantics as far as I understand it.